stages/org.osbuild.cpio.out

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+"""
+Assembles the tree into a CPIO archive named `filename`.
```

Comment:
Some other PR recently did the same wording, which is really confusing to me. It sounds like you literally mean `filename`.

Reply:
Dropped the last two words.